### PR TITLE
Appendix references and order

### PR DESCRIPTION
**Pull request by DamienIrving**
A number of the Appendices weren't referred to at all in text, so I've added references where needed.

I've also updated the order of the appendices so they appear in the order that they are first referenced in text.

### CODE REVIEW

File: _bookdown.yml [hunk 1]
```diff
@@ -28,15 +28,15 @@ rmd_files:
 
   # End files
   - appendix.Rmd
-  - glossary/glossary.md
+  - chapters/solutions.Rmd
   - chapters/objectives.Rmd
   - chapters/keypoints.Rmd
-  - chapters/solutions.Rmd
-  - chapters/anaconda.Rmd
   - chapters/tree.Rmd
```
Comment (k8hertweck) [inline _bookdown.yml:34]: I think style comes before tree?

Comment (DamienIrving) [reply _bookdown.yml:34]: There was a random/unnecessary reference to style in the project structure section, so I took it out and added a reference in the scripting chapter instead, which is where we first write some Python code and is thus the most appropriate place for the initial reference to that Appendix to be.

Comment (k8hertweck) [reply _bookdown.yml:34]: Got it, I should really stop trying to review when I'm in the middle of a hairy PR of my own.

**Review by k8hertweck — APPROVED** [review]
Thanks!

**Pull request merged**